contacts/contacts.service.js

```diff
@@ -0,0 +1,71 @@
+import {
+  data
+} from './data';
+import {
+} from './entities';
+import {
+  CONTACTS_SERVICE_OBJECT_NAME,
+} from './constants';
+import {
+  createPersonInstance,
+  localStorageService
+} from './helper';
+
+
+export class ContactsService {
+
+  constructor() {
+    this.internalArrayContacts = [];
+    this.init();
+  }
+
+  init = () => {
+    const localStorageObject = localStorageService.getFromLocalStorage(CONTACTS_SERVICE_OBJECT_NAME);
+    this.internalArrayContacts = (localStorageObject && localStorageObject.length) ? localStorageObject.map(item =>
```

Comment:
Please make this ternary operator easier to read. You can get the operations out in some variables.